### PR TITLE
added files for chinese translation

### PR DESCRIPTION
**Pull request by Manasa2850**
I've added these files:
devise.zh.yml
devise_invitable.zh.yml
zh.yml
kaminari.zh.yml

### CODE REVIEW

**Comment by Manasa2850** [comment]
@julianguyen I'll complete the translations soon!

**Comment by Manasa2850** [comment]
@julianguyen I've completed all the translations!

File: config/locales/zh.yml [hunk 1]
```diff
@@ -0,0 +1,728 @@
...
+      cookies: 'Cookies的使用'
+      cookies_description: '<p>该网站使用cookie来改善用户访问网站时的体验。在适用的情况下，本网站使用cookie控制系统，允许用户首次访问该网站时允许或禁止在其计算机/设备上使用cookie。</ p> <p> Cookie是保存给用户的小文件''的计算机硬盘驱动器，用于跟踪，保存和存储有关用户互动和网站使用情况的信息。这使网站可以通过其服务器为用户提供该网站内的量身定制的体验。</ p> <p>建议用户，如果他们希望拒绝使用和保存该网站上的cookie并将其保存到计算机硬盘上他们应在其网络浏览器的安全设置中采取必要的步骤，以阻止该网站及其外部服务供应商的所有cookie。</ p> <p>该网站使用跟踪软件来监视访问者，以更好地了解他们的使用方式。该软件由Google Analytics（分析）提供，它使用Cookie来跟踪访问者的使用情况。该软件会将Cookie保存到您计算机的硬盘中，以跟踪和监视您对网站的参与和使用情况，但不会存储，保存或收集个人信息。您可以阅读Google的％{google_privacy_policy}以了解更多信息。</ p>'
+      social_media: '社交媒体平台'
+      social_media_description: '<p>通过本网站参与的外部社交媒体平台进行的沟通，参与和采取的措施分别根据条款和条件以及每个社交媒体平台所持有的隐私政策而定制。</ p> <p>建议用户明智地使用社交媒体平台，并就其自身的隐私和个人详细信息谨慎/谨慎地与他们进行交流/互动。该网站绝不会通过社交媒体平台询问个人或敏感信息，并鼓励希望讨论敏感细节的用户通过电话或电子邮件等主要交流渠道与他们联系。</ p> <p>该网站可能会使用社交共享按钮这有助于直接从网页到相关的社交媒体平台共享Web内容。建议用户在使用此类社交共享按钮之前自行决定这样做，并请注意，社交媒体平台可以分别通过您的社交媒体平台帐户跟踪和保存您共享网页的请求。</ p>'
+      contact: '联系我们'
+      contact_description: '<p>如果有任何疑问或疑问，请％{email_link}。</ p> <p class =“ noMarginBottom”> <em>根据％{jamie_king_media}改编的隐私权政策。</ em> </ p>'
+    resources:
```
Comment (julianguyen) [inline config/locales/zh.yml]: There were changes made to this section of the translations! You'll want to rebase from master!

**Comment by Manasa2850** [comment]
@julianguyen I don't know why the tests are failing. When I tried to find out the difference in en.yml and zh.yml, I found that some of the keys were shown as different.
image
Like over here, 'empty_error' is also shown to be different in the 2 files.

**Pull request closed without merging**